src/fsharp/Fsc/project.json

```diff
@@ -1,13 +1,13 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23805",
```

Comment:
`Microsoft.NETCore.Platforms` it's already a dependency of `NETStandard.Library`. no need to add it